README.md

```diff
@@ -12,6 +12,14 @@ To communicate with the uart port of the SoC you need a usb to serial converter:
 * If you use the SmartRF06 board with an Evaluation Module (EM) mounted on it you can use the on-board ftdi chip. Make sure the "Enable UART" jumper is set on the board. You can have a look [here][contiki cc2538dk] for more info on drivers for this chip on different operating systems.
 * If you use a different platform, there are many cheap USB to UART converters available, but make sure you use one with 3.3v voltage levels.
 
+###Dependencies
+
+If you want to be able to program your device from an intel hex file, you will also need to install the following:
+
+* IntelHex: https://pypi.python.org/pypi/IntelHex
```

Comment:
Maybe add 'pip install intelhex' here? One less link to visit for the programmer that's less familiar with python.